README.md

```diff
@@ -31,12 +31,12 @@ https://github.com/open229/ruleset-model-description-schema
 The utility is intended to be used at a command line prompt:
 
 ```
-  createRulesetModelDescription in.epJSON
+  energyplus_create_rmd in.epJSON
```

Comment:
I would assume most users will use the command line interface in their workflows.  Once again, this would take a minor change to update to a binary named `energyplus_*`